examples/info_files/_INFO_file_template_minimal.json

```diff
@@ -11,17 +11,17 @@
 	"checkpoints": [
     {
       "name": "Source",
-      "processStartTime": "01-01-1970 00:00:00 +0000",
-      "processEndTime": "01-01-1970 0:00:00 +0000",
+      "processStartTime": "<ProcessStartTime>",
```

Comment:
I would make the minimal simpler, there are no data anyway. So that's why I think _"01-01-1970 00:00:00 +0000"_ is better here.